netci.groovy

```diff
@@ -1529,7 +1525,7 @@ combinedScenarios.each { scenario ->
                                         }
                                         else if (architecture == 'x64') {
                                             if (isLongGc(scenario)) {
-                                                buildCommands += "tests\\runtest.cmd ${runtestArguments} longgctests sequential"
+                                                buildCommands += "tests\\runtest.cmd ${runtestArguments} ${scenario} sequential"
```

Comment:
Just a suggestion can you move all the isLongGc checks before  https://github.com/dotnet/coreclr/pull/5621/files#diff-2d212ad84506951c8f928e21aed24580R1502

and then concatenate  ${scenario}  to ${runtestArguments}. Do you think something is preventing this ?